src/components/Avatar/Avatar.tsx

```diff
@@ -51,6 +51,14 @@ export const getInitials = (name?: string) => {
   return initials.join("");
 };
 
+export const checkImageOnLoadSuccessful = (src: string) => {
+  const img = new Image();
```

Comment:
Question; just trying to understand why this is needed. We already have the `onError` prop for handling errors. What does this prevent? Note; I think a lot of the re-rendering on the webapp side is an issue with graphql responses and caches, not the image component itself.

Reply:
reverting this